src/app/model/schema.ts

```diff
@@ -104,16 +103,16 @@ export enum MissionType {
   errand = "errand",
 }
 
-interface MissionDetails {}
+export interface MissionDetails {}
 
-interface FoodBoxDetails extends MissionDetails {
+export interface FoodBoxDetails extends MissionDetails {
   boxes?: Array<FoodBox>; // if multple boxes of same type grouping can occur in the UI
 }
 
 // created by the organiser.. but for MVP.0 we *hard code* a list of one type of box
-@Collection("foodboxes")
-export class FoodBox {
-  id!: string;
+// @Collection("foodboxes")
```

Comment:
We're not using the `@Collection` tags anymore right?

Reply:
removed